charts/service-binding-operator/README.md

```diff
@@ -0,0 +1,88 @@
+
+# Service Binding Operator Helm Chart
+
+This helm chart defines the Service Binding Operator. This provides an option for users to install  service binding operators using helm chart. 
+
+The chart installation will result in the creation of three Custom Resource Definitions(CRDs):
+- bindablekinds.binding.operators.coreos.com
+- servicebindings.binding.operators.coreos.com
+- servicebindings.servicebinding.io
+
+The resources required for service binding operator will also be installed. The chart creates a service-binding-operator namespace and installs the required resources in that namespace. 
+
+## Introduction
+
+The following are the values that can be customized when the chart is installed:
+
+- image.pullPolicy
+- keepTestResources
+
+A user can define values for the image  PullPolicy. As part of helm test we clear up the deploymemt,service binding resource and secret created. If a user is interested to view them , then he has to install the chart with keepTestResources set to true.
+
+## Helm Chart Installation 
+
+The helm chart installation involves the following steps:
+- Add the helm chart repository
+- Install the chart
+- Helm test
+
+Note: If service binding operator is not installed through OLM, the operator requires that cert-manager is available on the cluster. It can be installed by:
+
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.0/cert-manager.yaml
```

Comment:
The cert manager documentation refers not to embed cert-manager as a sub-chart of other Helm charts.We intended do it but dropped the idea because of that reason

Reply:
ok, but when speaking about Helm way to install, it would be good that all components are installed using Helm?

Reply:
I agree it would be good to have as a dependency and use helm dependency update to get latest version etc... However, I also understand the concern from cert manager team, if multiple charts make cert manager a dependency there could be a scenario where more than one cert manager is installed. I think it is worth exploring with them, cause I think our chart will fail if cert manager is already installed, which then will make our chart the bad actor. Either way, something to explore on future Epics, I would say.